skimage/exposure/tests/test_exposure.py

```diff
@@ -212,12 +212,12 @@ def test_adapthist_grayscale():
     img = rgb2gray(img)
     img = np.dstack((img, img, img))
     with expected_warnings(['precision loss|non-contiguous input']):
-        adapted = exposure.equalize_adapthist(img, 10, 9, clip_limit=0.01,
+        adapted = exposure.equalize_adapthist(img, (52, 64), clip_limit=0.01,
```

Comment:
As mentioned above, it's ok if the result is slightly different (if it was incorrect before), but it's not ok if the API to get such a result is different.